src/cts/src/CtsOptions.cpp

```diff
@@ -25,6 +25,18 @@ void CtsOptions::inDbInstCreate(odb::dbInst* inst, odb::dbRegion* region)
   recordBuffer(inst->getMaster(), getType(inst));
 }
 
+void CtsOptions::limitSinkClusteringSizes(unsigned limit)
+{
+  if (sinkClustersSizeSet_) {
+    setSinkClusteringSize(std::min(limit, sinkClustersSize_));
+    return;
+  }
+  auto lowerBound = std::lower_bound(
+      sinkClusteringSizes_.begin(), sinkClusteringSizes_.end(), limit);
+  sinkClusteringSizes_.assign(sinkClusteringSizes_.begin(), lowerBound);
```

Comment:
I don't think this is safe to do.  sinkClusteringSizes_.erase(lowerBound, sinkClusteringSizes_.end()); should work.  Are you intending to remove the lower_bound element and replace it with limit?